src/main/java/TaskToEventAuto.java

```diff
@@ -0,0 +1,54 @@
+package main.java;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class TaskToEventAuto implements TaskToEvent {
+    /**
+     *
+     * @param task      the task that needs to be converted into an event
+     * @param schedule  the schedule that the event will be added into
+     * @return an Event scheduled at a suggested time,
+     *          depending on the available times
+     */
+    @Override
+    public Event createEventFromTask(Task task, Schedule schedule) {
+        // if we have different schedules then the available time we get from for example the school schedule might be scheduled time in the family schedule
+        // potentially add a tag to events that will store the "schedule"
+
+        List<LocalDateTime> suggestedTimes = new ArrayList<>();
+        LocalDateTime availableTime = schedule.getAvailableTime(suggestedTimes, task.getDurationNeeded());
+        boolean scheduled = suggestTimeToUser(availableTime);
```

Comment:
maybe `confirmTimeWithUser`? "suggest" doesn't seem to imply that we want a response back (it could be purely informational to the user).